<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Gayathri" failures="0" tests="48" name="com.runner.RunnerClass" time="1982.345" errors="2" timestamp="2025-04-16T15:54:00 IST" skipped="0">
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="29.695"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="29.741"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="40.463"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="22.008"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="19.054"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="38.371"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="49.979"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="41.647"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="19.725"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="56.950"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="25.283"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="58.671"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="53.826"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="60.885"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="64.710"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="31.717"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="20.908"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="37.473"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="65.715"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="50.858"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="36.990">
    <error message="Step [the Doctor provides the discharge details] is defined with 1 parameters at &#039;com.definitions.PatientDischargeDefinition.the_doctor_provides_the_discharge_details(io.cucumber.datatable.DataTable)&#039;.
However, the gherkin step has 0 arguments.
Step text: the Doctor provides the discharge details" type="io.cucumber.core.exception.CucumberException">
      <![CDATA[io.cucumber.core.exception.CucumberException: Step [the Doctor provides the discharge details] is defined with 1 parameters at 'com.definitions.PatientDischargeDefinition.the_doctor_provides_the_discharge_details(io.cucumber.datatable.DataTable)'.
However, the gherkin step has 0 arguments.
Step text: the Doctor provides the discharge details
at io.cucumber.core.runner.PickleStepDefinitionMatch.arityMismatch(PickleStepDefinitionMatch.java:82)
at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:41)
at io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)
at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:81)
at io.cucumber.core.runner.TestStep.run(TestStep.java:53)
at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)
at io.cucumber.core.runner.TestCase.run(TestCase.java:84)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:75)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:135)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:136)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:136)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:132)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:81)
at org.testng.internal.thread.Async$AsyncTask.run(Async.java:34)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="60.858"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="60.654"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="37.936"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="40.464"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="19.760"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="60.228"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="42.066"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="21.606"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="65.420"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="64.787"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="40.463"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="64.733"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="31.667"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="12.882"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="49.546"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="21.844"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="48.054"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="23.416"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="24.273"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="79.233">
    <error message="Expected condition failed: waiting for frame to be available: __privateStripeFrame5024 (tried for 10 second(s) with 500 milliseconds interval)
Build info: version: &#039;4.30.0&#039;, revision: &#039;509c7f17cc*&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.6&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.85, chrome: {chromedriverVersion: 135.0.7049.95 (de2eb485a195..., userDataDir: C:\Users\gayat\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51775}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51775/devtoo..., se:cdpVersion: 135.0.7049.85, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 28a6572ea41fe4285980f07d146909e8" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for frame to be available: __privateStripeFrame5024 (tried for 10 second(s) with 500 milliseconds interval)
Build info: version: '4.30.0', revision: '509c7f17cc*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.6'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 135.0.7049.85, chrome: {chromedriverVersion: 135.0.7049.95 (de2eb485a195..., userDataDir: C:\Users\gayat\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51775}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51775/devtoo..., se:cdpVersion: 135.0.7049.85, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 28a6572ea41fe4285980f07d146909e8
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at com.definitions.PatientPaymentDefinitions.the_user_clicks_payment_option(PatientPaymentDefinitions.java:33)
at âœ½.the User clicks payment option(file:///C:/Users/gayat/git/Smart_Hospital_Cucumber/SmartHospitals_Cucumber/src/test/resources/Features/PatientPayment.feature:9)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="22.332"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="31.616"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="59.420"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="41.615"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="22.753"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="28.945"/>
  <testcase classname="com.runner.RunnerClass" name="runScenario" time="51.105"/>
</testsuite> <!-- com.runner.RunnerClass -->
